config/imsim.yaml

```diff
@@ -242,6 +235,9 @@ output:
 
     exp_time: 30
 
+    cosmic_ray_rate: 0.2
+    cosmic_ray_catalog:
```

Comment:
I think this is a syntax error.  To leave it empty, I'm pretty sure you need it to be `{}`.

Reply:
Actually, this is apparently fine. I just tried it, and it worked. Huh. Yaml is cleverer than I thought... :)

Reply:
Hmm.. that's surprising... I had to use the {} in the camera geometry case or it didn't work.

Reply:
I think with nothing there, YAML sets it to None, not an empty dict, which in this case is fine, but wouldn't be for camera_geometry.  In any case, I think it's probably cleaner to remove this line, since most users will be fine with the default CR file I suspect.

Reply:
I see.

So, if you used the {} would it overwrite the default so things wouldn't work?

In the old config we had 

```
catalog = default
```

which is at least good in letting people know the option is there.

Reply:
Right, but that nominally means to use the catalog file named `default`.  So IMO is more confusing than just leaving it out, which means to use the default value, which is some catalog we ship with imsim.

Reply:
Right.. I actually meant just to leave it blank with a comment as a reminder it is settable.  But, anyway I agree that almost no one will ever touch it.